packages/frint-component-handlers/src/RegionHandler.js

```diff
@@ -53,9 +53,10 @@ export default {
               return;
             }
 
-            const regionArgs = this.getProp('uniqueKey')
-              ? [this.getProp('name'), this.getProp('uniqueKey')]
-              : [this.getProp('name')];
+            const regionArgs = [
```

Comment:
I don't think we need to modify the Handlers for this. Just updating `frint-react` package should be enough.

Reply:
Ok, this was simpler than I thought.
I've removed the 2 commits related to `frint-component-handlers`